src/symbolic_dimensions.jl

```diff
@@ -1,26 +1,32 @@
 import .Units: UNIT_SYMBOLS, UNIT_MAPPING, UNIT_VALUES
 import .Constants: CONSTANT_SYMBOLS, CONSTANT_MAPPING, CONSTANT_VALUES
 
+
 const SYMBOL_CONFLICTS = intersect(UNIT_SYMBOLS, CONSTANT_SYMBOLS)
 
+disambiguate_symbol(s) = s in SYMBOL_CONFLICTS ? Symbol(string(s) * "_constant") : s
```

Comment:
I think the following is sufficient:
```suggestion
disambiguate_symbol(s) = s in SYMBOL_CONFLICTS ? Symbol(s, :_constant) : s
```

Reply:
Maybe it would be better to avoid these conflicts in the first place though, e.g. by keeping separate containers or using a `_constant` suffix for all constants.

Reply:
Good tip, I didn't know that syntax!

> Maybe it would be better to avoid these conflicts in the first place though, e.g. by keeping separate containers or using a `_constant` suffix for all constants.

Oh, the `_constant` here is just for printing. The `h` symbol still works as you might expect, in a separate module:

```julia
julia> using DynamicQuantities.SymbolicUnits: h as h1

julia> using DynamicQuantities.SymbolicConstants: h as h2

julia> h1
1.0 h

julia> h2
1.0 h_constant
```

Then in string form you would write `us"Constants.h"`